gen.py

```diff
@@ -0,0 +1,26 @@
+from argparse import ArgumentParser
+from pathlib import Path
+
+import qmkcommentgen
+
+
+if __name__ == "__main__":
+    parser = ArgumentParser(description='Generate QMK firmware keymap comments')
+    parser.add_argument('keymappath', type=str, help='Path to keymap.c')
```

Comment:
```suggestion
    parser.add_argument('-i','--keymap-path', type=str, default='./', help='Path to keymap.c')
```

Reply:
This makes more sense and allows the python file to run without opening the command line